packages/caliper-fabric/lib/identity-management/IdentityManager.js

```diff
@@ -191,7 +191,14 @@ class IdentityManager {
      * @private
      */
     async _extractIdentitiesFromWallet(mspId, wallet) {
-        // TODO: To be implemented
+        const walletFacade = await this.walletFacadeFactory.create(wallet.path);
+        const allIDNames = await walletFacade.getAllIdentityNames();
+        // eslint-disable-next-line no-console
+        console.log({allIDNames});
```

Comment:
Looks like you left some debug code in, could you delete the console.log and the comment above it as well.